examples/pulse_control/6_Use_jobs_with_pulse_sequence_sweeps.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<h1>Use jobs for a frictionless experience with pulse sequence sweeps</h1><p> </p><p>... Extracting this experimental information will allow for building a device Hamiltonian model that could then be compared with results obtained from the real device. </p><p> </p><p>Often, protocols to obtain this experimental data involves varying a control parameter and recording the response of the system. <code>This was for instance the case in the first example notebook</code>. -<strong>&gt; Link to the notebook instead</strong> </p><p>Here we will revisit this notebook by taking advantage of Hybrid Jobs. We will encapsulate a frequency sweep in a single job instance, which will give tasks created from within the job priority access to the device.</p><p>You can learn more about Hybrid Jobs <a href="https://github.com/aws/amazon-braket-examples/raw/161b36cc401132f5baba3dc9f43db16503bfdbca/examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/Creating_your_first_Hybrid_Job.ipynb" target="_blank">here</a> and how to use them to implement hybrid classical-quantum algorithms in a production environment. </p><p>You can use either Rigetti's Aspen M-3 or OQC's Lucy device to run this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>